compressed_depth_image_transport/src/codec.cpp

```diff
@@ -96,8 +97,8 @@ sensor_msgs::Image::Ptr decodeCompressedDepthImage(const sensor_msgs::Compressed
       }
       catch (cv::Exception& e)
       {
-        ROS_ERROR("%s", e.what());
-        return sensor_msgs::Image::Ptr();
+        RCUTILS_LOG_ERROR(e.what());
```

Comment:
I switched these to `RCLCPP_ERROR` per @mikaelarguedas suggestion.

Reply:
I [did that](https://github.com/ros-perception/image_transport_plugins/pull/27/files#diff-0040405190ff58f2bcddd94955a8a0a7R71) for the code encapsulated in classes but I doubt about doing it in the codec file that contains helper functions. We can modify the interface to include a node shared ptr but looks a bit weird to me to do that for "decode" or "encode" functions.

Reply:
Yeah, I'm still a little unclear on what the best practice is here, especially since logging requires passing either a `Node` or `Logger` around.

Reply:
On `gazebo_ros_pkgs` sometimes we create a logger on the fly, i.e.

`auto logger = rclcpp::get_logger("compressed_depth_image_transport");`

Maybe here it makes sense to have a static logger.